tests/unit/test_estimator.py

```diff
@@ -157,6 +162,7 @@ def sagemaker_session():
                                                       return_value=DESCRIBE_TRAINING_JOB_RESULT)
     sms.sagemaker_client.describe_endpoint = Mock(return_value=ENDPOINT_DESC)
     sms.sagemaker_client.describe_endpoint_config = Mock(return_value=ENDPOINT_CONFIG_DESC)
+    sms.sagemaker_client.list_tags = Mock(return_value=LIST_TAGS_RESULT)
```

Comment:
the mock call is set up here, but there's no test that shows estimator uses the result of the call. adding an assert for this to `test_attach_framework` would cover this.